analyticsdataserver/settings/test.py

```diff
@@ -14,11 +14,27 @@
         "HOST": "",
         "PORT": "",
     },
+    'analytics': {
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
I just realized these may need to be different. I don't think so, given that I get v1 specific database query errors but no v0 errors, but I'm not sure. I'll try to look into it.

Reply:
I can't find information supporting or refuting this. I'll just change the name for the sake of clarity, at the very least. I'll push up the change once first pass reviews are done.